### ISSUE TITLE
change default style ext

### ISSUE THREAD

**Issue by lovesora** [angular/angular-cli#11032]
### Versions
```
Angular CLI: 6.0.0
Node: 8.11.2
OS: darwin x64
Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.6.0
@angular-devkit/core         0.6.0
@angular-devkit/schematics   0.6.0
@schematics/angular          0.6.0
@schematics/update           0.6.0
rxjs                         6.2.0
typescript                   2.7.2
```

### Repro steps

* exec `ng config schematics.@schematics/angular:component.styleext scss -g` as the official doc said [stories-css-preprocessors](https://github.com/angular/angular-cli/wiki/stories-css-preprocessors)

### Observed behavior
```shell
Invalid JSON character: "s" at 0:0.
Error: Invalid JSON character: "s" at 0:0.
    at _readValue (/usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/core/src/json/parser.js:567:19)
    at parseJsonAst (/usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/core/src/json/parser.js:605:17)
    at Object.parseJson (/usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/core/src/json/parser.js:631:12)
    at normalizeValue (/usr/local/lib/node_modules/@angular/cli/commands/config.js:128:19)
    at ConfigCommand.set (/usr/local/lib/node_modules/@angular/cli/commands/config.js:195:23)
    at ConfigCommand.run (/usr/local/lib/node_modules/@angular/cli/commands/config.js:168:18)
    at /usr/local/lib/node_modules/@angular/cli/models/command-runner.js:278:30
    at Generator.next (<anonymous>)
    at fulfilled (/usr/local/lib/node_modules/@angular/cli/models/command-runner.js:4:58)
    at <anonymous>
```

### Desired behavior
don't throw error

**Comment by lovesora** [comment]
but when i exec `ng new scss-project --style=scss`, it works

**Comment by mixtou** [comment]
I have the same problem when trying to change existing project from css to scss by executing:
`ng config defaults.styleExt scss`

`     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 6.0.5
Node: 10.1.0
OS: darwin x64
Angular: 6.0.3
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.3
@angular-devkit/build-angular     0.6.3
@angular-devkit/build-optimizer   0.6.3
@angular-devkit/core              0.6.3
@angular-devkit/schematics        0.0.46
@angular/cdk                      6.1.0
@angular/cli                      6.0.5
@angular/flex-layout              6.0.0-beta.15
@angular/material                 6.1.0
@ngtools/json-schema              1.1.0
@ngtools/webpack                  6.0.3
@schematics/angular               0.6.5
@schematics/schematics            0.0.13
@schematics/update                0.6.5
rxjs                              6.2.0
typescript                        2.7.2
webpack                           4.8.3`

**Comment by clydin** [comment]
Resolved via #10956 and will be in the next patch release.
